spec/services/stats/r102_sar_appeals_performance_report_spec.rb

```diff
@@ -74,6 +75,12 @@ module Stats
         create_ico(type: :sar, received: '20170604', responded: '20170629', deadline: '20170625', team: @team_c, responder: @responder_c, ident: 'ico sar for team c - responded late')
         create_ico(type: :sar, received: '20170606', responded: '20170625', deadline: '20170630', team: @team_c, responder: @responder_c, ident: 'ico sar for team c - responded in time')
         create_ico(type: :sar, received: '20170605', responded: nil,        deadline: '20170702', team: @team_d, responder: @responder_d, ident: 'ico sar for team d - open in time')
+        
+        # SAR Internal Reviews
+        create_sar_ir(received: '20170605', deadline: '20170702', team: @team_d, responder: @responder_d, ident: 'case for team d - open in time', case_type: :closed_sar_internal_review)
+        create_sar_ir(received: '20170606', deadline: '20170630', team: @team_c, responder: @responder_c, ident: 'case for team c - responded in time', case_type: :closed_sar_internal_review)
+        create_sar_ir(received: '20170607', deadline: '20170625', team: @team_b, responder: @responder_b, ident: 'case for team b - responded in time', case_type: :closed_sar_internal_review)
+        create_sar_ir(received: '20170606', deadline: '20170630', team: @team_a, responder: @responder_a, ident: 'case for team a - responded in time', case_type: :closed_sar_internal_review)
```

Comment:
it doesn't seem to cover other possible category like like responded-late, open-in-time, open-late,  but this only minor issue and can be improved later, shouldn't prevent it from releasing this feature